modules/users/client/components/UserLink.js

```diff
@@ -7,14 +8,21 @@ import React from 'react';
  * @param {string} user.displayName
  * @param {string} user.username
  */
-export default function UserLink({ user }) {
+export default function UserLink({ user, className }) {
+  const { t } = useTranslation('users');
+
+  if (!user?.username) {
+    return <span className={className}>{t('Anonymous member')}</span>;
```

Comment:
"New feature". Just to protect against cases where we don't have username when user has been deleted.